packages/visx-grid/test/GridPolar.test.tsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 
+import { scaleLinear } from '@visx/scale';
 import { GridPolar, GridAngle, GridRadial } from '../src';
-import { scaleLinear } from '../../visx-scale';
```

Comment:
this will cause tests to fail when visx-scale in the upper folder is not built

Reply:
great catch. this used to happen in several packages and was mostly cleaned up in the TS re-write but clearly missed some 🙉